kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunner.kt

```diff
@@ -63,18 +68,38 @@ interface WorkflowRunner<out OutputT> {
       activity: FragmentActivity,
       viewRegistry: ViewRegistry,
       workflow: Workflow<InputT, OutputT, Any>,
-      inputs: Flowable<InputT>,
+      inputs: () -> ReceiveChannel<InputT>,
```

Comment:
Should document here that we will tear down the channel, no? Could go into this being a proxy for a cold observable, etc.

Reply:
- [ ] add docs